### ISSUE TITLE
[Feature request] Make hiking/cycling routes highlightable

### ISSUE THREAD

**Issue by knusprjg** [osmandapp/Osmand#3969]
I noticed that it is pretty hard to follow hiking/cycling routes on the map, in particular when there are many other routes around that share the same path. In this case, always only one color dominates, while the other(s) suddenly disappear, making it hard to keep track. I attached a screenshot where the blue route goes missing as soon as it crosses the pink route.

What I would like to propose is that hiking/cycling tracks, or even bus lines etc. could be made highlightable as if they were GPX tracks. Currently hiking routes are treated like POIs, so when I search for them, only the orange tag appears on one point of the route (see screenshot lower right). It would be nice if the whole track would be highlighted instead, or at least that the option to highlight the track is given in the pop-up menu. In my opinion this would a great improvement to an already great feature :)

![hiking_track](https://user-images.githubusercontent.com/23284327/27518761-8677f780-59e7-11e7-9aa9-25c9e2282d91.png)

**Comment by scaidermern** [comment]
With the recent improvements in public transport navigation it should be possible to implement this, too.

It would be great to make hiking and cycling routes selectable. Tapping on a route symbol should open a widget with route details and highlight the whole route, similar to public transport routes. It should also be possible to navigate along this route, similar to a GPX track.

This will also mitigate the shortcomings of #1721.

**Comment by vshcherb** [comment]
Merged into #7299
